envoy/api/v2/route/route.proto

```diff
@@ -564,6 +564,15 @@ message RedirectAction {
 
   // [#not-implemented-hide:] The scheme portion of the URL will be swapped with "https".
   bool https_redirect = 4;
+
+  // [#not-implemented-hide:] Indicates that during redirection, the matched prefix (or path) should
```

Comment:
What is meant by "or path" here?

Reply:
By path I meant that the match condition can also be a path based match ( not just prefix ).